components/server/src/server.ts

```diff
@@ -297,6 +297,9 @@ export class Server<C extends GitpodClient, S extends GitpodServer> {
     }
 
     protected async startLongRunningMigrations(): Promise<void> {
+        if (this.config.disableLongRunningMigrationsJob) {
```

Comment:
```suggestion
        if (this.config.longRunningMigrationsJob.disabled) {
```
should it be like this?